src/stirling/source_connectors/socket_tracer/protocols/mongodb/types.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "src/stirling/source_connectors/socket_tracer/protocols/common/event_parser.h"
+#include "src/stirling/utils/utils.h"
+
+namespace px {
+namespace stirling {
+namespace protocols {
+namespace mongodb {
+
+enum class Type : int32_t {
+  kOPMsg = 2013,
+  kOPReply = 1,
```

Comment:
While this isn't in our style guide, all of the existing enums and enum classes are sorted by value. Therefore, I think it's best to maintain that consistency and move this after `kOPCompressed = 2012,`.